readme.md

```diff
@@ -245,6 +245,7 @@
 
 ## Back-End Development
 
+- [Django](https://github.com/shahraizali/awesome-django)
```

Comment:
> The entry in the Awesome list should:
Include a short description about the project/theme of the list. It should not describe the list itself.

You also need the `#readme` part.